src/Domain/Entities/Payment.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    public class Payment
```

Comment:
Ten obiekt nie ma na ten moment sensu. Jak zamodelujemy odpowiednio zamówienie itd, to będziemy myśleć jak to zrobić sensowniej